ui/src/components/pages/Pipeline/partials/Graph/partials/NodeInfo/partials/Pods/partials/PodDetails/partials/Metrics/partials/LineChart/index.tsx

```diff
@@ -373,9 +374,21 @@ const LineChartComponent = ({
 
     const labels: string[] = [];
     const transformedData: Record<string, any>[] = [];
+    let filteredChartData = chartData;
     const label = groupByLabel(metricsReq?.dimension, metricsReq?.display_name);
 
-    chartData?.forEach((item) => {
+    if (JSON.stringify(label) === JSON.stringify(["container"])){
```

Comment:
lets have a stricter comparison like this - `if (Array.isArray(label) && label.length === 1 && label[0] === "container")`
stringify might be an overkill so this simple conversion.